specification/hybridcloud/resource-manager/Microsoft.HybridCloud/preview/2023-01-01-preview/hybridCloud.json

```diff
@@ -0,0 +1,1003 @@
+      "type": "object",
+      "properties": {
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Resource tags."
+        }
+      },
+      "description": "Tags object for patch operations."
+    },
+    "CloudConnectorListResult": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/CloudConnector"
+          },
+          "x-ms-identifiers": [
+            "id"
+          ],
+          "description": "A list of cloud connector resources."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "Link to hold the next page of results"
+        }
+      },
+      "description": "Response for the ListCloudConnectors API service call"
+    },
+    "CloudConnectorPropertiesFormat": {
+      "type": "object",
+      "properties": {
+        "accountId": {
+          "type": "string",
+          "description": "Account identifier of the remote cloud."
+        },
+        "cloudType": {
+          "type": "string",
+          "description": "The cloud connector type.",
+          "enum": [
+            "AWS"
+          ],
+          "x-ms-enum": {
+            "name": "cloudType",
+            "modelAsString": true
+          }
+        },
+        "provisioningState": {
+          "readOnly": true,
+          "$ref": "#/definitions/ProvisioningState",
+          "description": "The provisioning state of the cloud connector resource."
+        }
+      },
+      "description": "Cloud connector resource properties."
+    },
+    "CloudConnector": {
+      "type": "object",
+      "x-ms-azure-resource": true,
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/CloudConnectorPropertiesFormat",
+          "description": "Properties of the cloud connector."
+        },
+        "etag": {
+          "type": "string",
+          "readOnly": true,
+          "description": "A unique read-only string that changes whenever the resource is updated."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/TrackedResource"
+        }
+      ],
+      "description": "Cloud Connector resource."
+    },
+    "CloudConnectionListResult": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/CloudConnection"
+          },
+          "description": "A list of cloud connection resources."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "Link to hold the next page of results"
+        }
+      },
+      "description": "Response for the ListCloudConnections API service call"
+    },
+    "CloudConnectionPropertiesFormat": {
```

Comment:
Can it just be called CloudConnectionProperties instead of CloudConnectionPropertiesFormat?

Reply:
done